roles/openshift_logging/templates/es.j2

```diff
@@ -95,6 +95,13 @@ spec:
               readOnly: true
             - name: elasticsearch-storage
               mountPath: /elasticsearch/persistent
+          readinessProbe:
+            exec:
+              command:
+              - "/usr/share/elasticsearch/probe/readiness.sh"
+            initialDelaySeconds: 5
+            timeoutSeconds: 5
```

Comment:
@ewolinetz @jcantrill @richm I am going to remove the `timeoutSeconds` from DeploymentConfig in this PR. This means we will have to rely on timeout implemented directly in the check itself, which is by default set to 1 sec.